server/controllers/apply/applicationsController.test.ts

```diff
@@ -39,7 +39,7 @@ describe('applicationsController', () => {
 
   describe('list', () => {
     it('renders the list view', async () => {
-      applicationService.tableRows.mockResolvedValue([])
+      applicationService.dashboardTableRows.mockResolvedValue([])
```

Comment:
Not for now, but I think it might be worth us moving a bunch of this kind of logic to helper functions that we can use in the views, rather than sending marshalled data via the controller. Just so we're seperating data and presentation layer concerns a bit better

Reply:
I'm quite aware that I might be breaking this rule elsewhere (and I probably introduced the patten in the first place 😜 )